ouroboros-consensus/src/Ouroboros/Consensus/HeaderValidation.hs

```diff
@@ -31,6 +31,7 @@ module Ouroboros.Consensus.HeaderValidation (
   , HeaderState (..)
   , castHeaderState
   , genesisHeaderState
+  , headerStateBlockNo
```

Comment:
Why is this needed? Maybe it's worth adding a comment in the function.

Reply:
`headerStateBlockNo` is a convenience accessor much like `headerStatePoint` just below.

It was requested by @dcoutts to simplify the implementation of `answerQuery` https://github.com/input-output-hk/ouroboros-network/blob/0a341b18f414bf21e426bc0090dd432c0121aa97/ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs#L284

Originally I had used a case expression so looking at it now the its `fmap` form, inlining the function into `answerQuery` is not so bad.

Reply:
I see. Thanks.